.github/workflows/test.yml

```diff
@@ -25,3 +25,16 @@ jobs:
       - name: Test
         run: .circleci/test.sh
         timeout-minutes: 15
+
+      - name: Retrieve screenshots
+        if: failure()
+        run: .circleci/process-artifacts.sh
```

Comment:
is process-artifacts.sh the correct name for this? should it not be renamed to match what it does?

Reply:
It's for copying screenshots out of the container and into the CI workspace. I think it's fine.

It's also consistent with a dozen other repos at this point.